src/from-scratch.js

```diff
@@ -1,16 +1,39 @@
-const resolvedWrapper = () => {
+const resolvedWrapper = (value) => {
+  return Promise.resolve(value);
```

Comment:
Can you instead show how to do this using `new Promise()` and then include this as a comment with a note along the lines of "did you know you could also solve it like this?" and explain what Promise.resolve() does

The reason is that I include the `new Promise()` syntax in my notes and didn't include `Promise.resolve`